src/legacy/core_plugins/tagcloud/public/__snapshots__/tag_cloud_fn.test.js.snap

```diff
@@ -9,14 +9,13 @@ Object {
       "listenOnChange": true,
     },
     "visConfig": Object {
+      "bucket": undefined,
```

Comment:
Is this safe? And is it safe for `metric: undefined` as well? You don't check for a value of either property before putting it on the `visConfig` object in `tag_cloud_fn.js`, so that can end up as the output.

Reply:
should be safe, but i will still remove this, better if its not set at all